ext/bindings/message.cpp

```diff
@@ -43,9 +50,10 @@ void bind_message(Module& module) {
     ;
 
   define_class_under<pulsar_rb::Message>(module, "Message")
-    .define_constructor(Constructor<pulsar_rb::Message, const std::string&>())
+    .define_constructor(Constructor<pulsar_rb::Message, const std::string&, const std::string&>())
     .define_singleton_method("from_message", &pulsar_rb::Message::fromMessage)
     .define_method("data", &pulsar_rb::Message::getData)
     .define_method("message_id", &pulsar_rb::Message::getMessageId)
+    .define_method("partition_key", &pulsar_rb::Message::getPartitionKey)
```

Comment:
do we care about a complementary `partition_key=` method? i.e. extract a method from lines 18-20, maybe with an else clause for clearing the partition key if empty. then expose it here. we could wire up a ruby facade that translates nil -> "" before passing it to CPP so `msg.partition_key = nil` in ruby-land can work

Reply:
sure I think that would be a great idea, and I think translating nil to "" is good too

Reply:
does that imply that we should also include a `data=` method as well?